plug.vim

```diff
@@ -763,25 +764,27 @@ function! s:update_impl(pull, force, args) abort
   call s:prepare()
   call append(0, ['', ''])
   normal! 2G
+  redraw
```

Comment:
@junegunn I believe I used that line when I first started porting & the python was still crashing. I was checking the side pane code worked. The side pane created doesn't appear until a redraw is issued in neovim. Not necessary now that everything works.